packages/base-types/src/models/project.ts

```diff
@@ -27,6 +27,8 @@ export interface ProjectPrototype {
   trainedModel?: PrototypeModel;
   lastTrainedTime?: number;
   nlp?: ProjectPrototypeNLP;
+  delayEnabled: boolean;
+  delay: number;
```

Comment:
based on designs, we want these to be separate https://app.zeplin.io/project/5cfebcae362911161ad0c2ca/screen/61db2941e7d9b62c3e3681fa

Reply:
these props are for global settings